elasticapm/instrumentation/packages/asyncio/base.py

```diff
@@ -29,8 +29,33 @@
 #  OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 from elasticapm.instrumentation.packages.base import AbstractInstrumentedModule
+from elasticapm.traces import execution_context
+from elasticapm.utils.wrapt import function_wrapper
 
 
 class AsyncAbstractInstrumentedModule(AbstractInstrumentedModule):
     async def call(self, module, method, wrapped, instance, args, kwargs):
         raise NotImplementedError()
+
+
+class RunInExecutorInstrumentation(AsyncAbstractInstrumentedModule):
+    name = "executor"
+    instrument_list = [("asyncio.base_events", "BaseEventLoop.run_in_executor")]
```

Comment:
Might we worth checking if this would also work with uvloop (https://github.com/MagicStack/uvloop/blob/8beacd265651429a558d1b22e200d6d81016f1db/uvloop/loop.pyx#L2646), but I have my doubts, as wrapt can't patch objects defined in C AFAIK